### DISCUSSION TITLE
Apk size

### DISCUSSION THREAD

**Discussion by J4CKED** [FossifyOrg/General-Discussion#391] [Q&A] [answered]
Why has the app grown so much?
Version  1.1.3 - 16.5 MB
Version 1.2.1 - 42.1 MB

**Comment by naveensingh** [comment, marked as answer]
That is due to JPEG XL support. See #3 for more info. 

Related: https://www.reddit.com/r/Fossify/comments/1fo7s2m/why_is_the_apk_for_fossify_gallery_120_over_twice/
